addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -148,10 +148,17 @@
                 </DatiBollo>
                 <ImportoTotaleDocumento t-esc="format_monetary(document_total, currency)"/>
             </DatiGeneraliDocumento>
-            <DatiOrdineAcquisto t-if="record.ref">
+            <DatiOrdineAcquisto t-if="record.ref and not record.reversed_entry_id">
                 <IdDocumento t-esc="format_alphanumeric(record.ref[:20])"/>
             </DatiOrdineAcquisto>
-            <DatiDDT t-if="record.l10n_it_ddt_id">
+            <DatiFattureCollegate t-if="record.reversed_entry_id">
+                <IdDocumento t-esc="format_alphanumeric(record.reversed_entry_id.name[-20:])"/>
+                <Data t-if="record.reversed_entry_id.invoice_date" t-esc="format_date(record.reversed_entry_id.invoice_date)"/>
```

Comment:
```suggestion
                <Data t-out="format_date(record.reversed_entry_id.invoice_date)"/>
```
Maybe t-out to have less conflicts in the fw-port and afaik the field is required.

Reply:
Please check again

Reply:
@jco-odoo  👀 sorry!  
(WTF??? Did I lose a commit or a push somewhere?)